observations/obs_converters/GSI2DART/enkf/readconvobs.f90

```diff
@@ -16,7 +16,7 @@ module readconvobs
 !
 ! program history log:
 !   2009-02-23  Initial version.
-!   2019-08-26  A little more error handling. Using open_file()
+!   2020-08-26  A little more error handling. Using open_file()
```

Comment:
i don't see the change that uses open_file().  are you missing a changed file in this pull request?

Reply:
I have pushed several times to this pull request ... so I am confused by what you are seeing - or more correctly - what github is showing as 'differences'.  If the code has open_file() **at all**, you are seeing the 'latest' version.